app/components/Application/ApplicationDetailsCardItemCustomFields.tsx

```diff
@@ -4,17 +4,28 @@ import SummaryEmissionSourceFields from 'containers/Forms/SummaryEmissionSourceF
 import ProductionFields from 'containers/Forms/ProductionFields';
 import {FieldProps} from 'react-jsonschema-form';
 import NumberFormat from 'react-number-format';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faExclamationTriangle} from '@fortawesome/free-solid-svg-icons';
 
 const customFields = (
   showDiff,
   diffPathArray,
   diffArray,
   handleEnums,
-  previousIsEmpty
+  previousIsEmpty,
+  setHasErrors
 ) => {
   const CUSTOM_FIELDS: Record<string, React.FunctionComponent<FieldProps>> = {
     TitleField: props => <h3>{props.title}</h3>,
     StringField: props => {
+      let hasErrors;
```

Comment:
rename to `errorIcon` maybe, and give it a type? I would expect `hasErrors` to be a boolean